stream_alert/alert_processor/outputs.py

```diff
@@ -101,24 +100,24 @@ def dispatch(self, **kwargs):
             'rule_description': rule_desc,
             'record': kwargs['alert']['record']
         }
-        values_json = json.dumps({
+        data = {
             'service_key': creds['service_key'],
             'event_type': 'trigger',
             'description': message,
             'details': details,
             'client': 'StreamAlert'
-        })
+        }
+        headers = None
+        verify = True
 
-        resp = self._request_helper(creds['url'], values_json)
+        resp = self._post_request(creds['url'], data, headers, verify)
```

Comment:
Suggestion: don't instantiate the `headers` and `verify` vars above just to pass them to this function - just pass them directly since they aren't used anywhere else in this functions scope.